README.md

```diff
@@ -102,3 +112,49 @@ mnc2nii PATH_TO/avg.XXX.mnc PATH_TO/template_XXX.nii
 ~~~
 
 </details>
+
+### Steps to register a subject to the T2w dog template. 
+
+We picked `sub-HarshmanDobby` for this.
+
+#### Data
+
+**[with internal data access]** Required data is at the `data.neuro.polymtl.ca:datasets/template_dog_virginiatech` repository under the `rb/update_HB` branch
+
+**[with no internal data access]** Please download the below data, with only sub-HarshmanDobby and it's derivatives (contains no sensitive information)
+[reg_to_temp_bids.zip](https://github.com/user-attachments/files/16071719/reg_to_temp_bids.zip)
+
+1. Protocol to segment the spinal cord of the sub-HarshmanDobby
+```
+Command used : sct_deepseg_sc -i sub-HarshmanDobby_T2w.nii.gz -c t2 -o sub-HarshmanDobby_T2w_label-SC_seg.nii.gz
+Manual correction was done by Rohan Banerjee on the output from sct_deepseg_sc
+```
+
+2. Protocol to get the disc labels
+```
+sct_label_utils -i sub-HarshmanDobby_T2w.nii.gz -create-viewer 1:24 -o sub-HarshmanDobby_T2w_labels-disc.nii.gz
+```
+
+The following steps were done for the registration to the template (change parameters accordingly):
```

Comment:
> change parameters accordingly

according to what?

Reply:
I wanted to convey that in case you are using some other subjects (other dataset) then change the parameters accordingly. But I guess I don't to mention that there since we are just showing the registration of `sub-HarshmanDobby`

Reply:
Updated here: https://github.com/spinalcordtoolbox/template-dog/commit/7076ccecdae1c92fbc1218097a5c691bde642938